robot/Cumulus/resources/OpportunityPageObject.py

```diff
@@ -17,8 +17,7 @@ def _is_current_page(self):
         """
         self.selenium.wait_until_location_contains("/lightning/r/Opportunity/",timeout=60,message="Current page is not a Opportunity detail view")
         self.selenium.wait_until_page_contains("Donation Information")
-        locator = npsp_lex_locators['manage_hh_page']['more_actions_btn']
-        self.selenium.wait_until_element_is_visible(locator,60)
+        self.selenium.wait_until_page_contains("Delete")
```

Comment:
Seems like 2 different locators to me. Originally it was trying to find Manage HH and now Delete. Please make sure if this keyword is being used anywhere else and solving the same issue.

Reply:
Great. Look good then.